src/containers/Bookings.js

```diff
@@ -1,19 +1,62 @@
-import React, { Component } from 'react';
-import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import React, { Component } from "react";
+import Search from "../components/Search.js";
+import BookingsMessage from "../components/BookingsMessage.js";
+import Results from "../components/Results.js";
+import fakeBookings from "../data/fakeBookings.json";
+import Counter from "../components/Counter";
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info('TO DO!');
+  state = {
+    count: 0,
```

Comment:
I am not sure what count1, count2 are? can they have better names to describe what they do? same for `input`.

Reply:
This is just rubbish, it was just for myself. I just left them there for a future reference.